### PR TITLE
change dependency of useEffect() hook in repos component from props.r…

### LINKED ISSUE #10: Solution: Repos component's useEffect() isn't set up with the right prop name.
the Card component passes in a property 'url' to the repos component
```
{repos_url ? <Repos url={repos_url} /> : null}
```
but it is referred to in useEffect() , in the Repos component as props.repos (line 11).
This should be props.url
```
  React.useEffect(() => {
    getData(props.url).then(repos => setRepos(repos));
  }, [props.repos]);
```

### PR DESCRIPTION
**Pull request by rooberrydev**
…epos to props.url resolves #10

this was slightly confusing to FAC-17 as react hook dependencies were being explained, but the dependency was incorrectly named in the solution file.

### CODE REVIEW

File: solution/src/components/card/index.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ function Card() {
     public_repos,
     repos_url,
     login,
-    url,
+    url
```
Comment (oliverjam) [inline solution/src/components/card/index.js]: I don't think this change was necessary (I assume your Prettier config has trailing commas turned off)

```suggestion
    url,
```

**Review by oliverjam — CHANGES_REQUESTED** [review]

File: solution/src/components/card/index.js (hunk 1 above)
Comment (rooberrydev) [reply solution/src/components/card/index.js]: oh yeah that was definitely prettier not me eeks

Comment (sofiapoh) [reply solution/src/components/card/index.js]: if you commit the suggestion @oliverjam made I can merge this :)

Comment (rooberrydev) [reply solution/src/components/card/index.js]: @sofiapoh I have fixed the trailing comma, sorry it took so long, this got lost in my notifications

**Pull request merged**